### PR TITLE
Adds a blog entry for updating IndigoSandbox to IndigoGame

### PR DESCRIPTION
**Pull request by hobnob**

### CODE REVIEW

File: website/blog/2022-01-31-hello-indigo-game.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# Converting `IndigoSandbox` to `IndigoGame`
```
Comment (davesmith00000) [inline website/blog/2022-01-31-hello-indigo-game.md:1]: You'll need to add the headers, e.g.:
```
---
title: Post-Mortem: "RoguelikeDev Does The Complete Roguelike Tutorial" in Scala
author: Dave Smith
authorURL: http://twitter.com/davidjamessmith
---
```
I'm sure docusaurus will have a "draft" flag too.

**Comment by davesmith00000** [comment]
Great stuff, as I said in our chat, it'll be great to start getting blog posts up. 😄 

Some observations:

- You’ve got a few minor wording errors, just needs a re-read with fresh eyes.
- I’d be careful to couch it as a minimal change from Sandbox to Game since you’re not actually going to use scenes.
- Personally I’d flip the `EventFilter` information, start them on permissive and recommend restricted - otherwise the readers who are likely beginners will be wondering why their events aren’t showing up. Optimise later, make it easy for them for now.
- The stuff about referring to “(lines 12-13, 22-23, and 25-26)“, you should either put in a link to the code, or maybe just show the diff of you removing that code?
- Actually doing this as a series of diffs (which docusaurus will render nicely) isn’t a bad idea maybe?

**Comment by davesmith00000** [comment]
Random thought, should this be a blog post or one of the how-to guides? (Which we could still promote)

Blog posts for an engine to me feel like they ought to be "of their time"? Where as this feels like something you might want to maintain with new releases.

What do you think?

**Comment by hobnob** [comment]
> Random thought, should this be a blog post or one of the how-to guides? (Which we could still promote)
> 
> Blog posts for an engine to me feel like they ought to be "of their time"? Where as this feels like something you might want to maintain with new releases.
> 
> What do you think?

Yes I was wondering the same thing - definitely worth moving to `Guides` I think

File: indigo/docs/guides/howto-indigo-game.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+You may have noticed that many of our tutorials and examples make use of
+`IndigoSandbox` and `IndigoDemo`. But what if you want to go further than these
+examples? How would you add scene management, or deal with sub systems? In these
+cases it's time to move your game to use `IndigoGame`, which comes with a host
+of features, but also a lot more boiler-plate code.
+
+Note: In this tutorial we'll be moving our code to `IndigoGame` and using an
+empty scene, but we won't be making full use of the scene management.
+If you want to know more about managing scenes in Indigo you can read the
+documentation [here](https://indigoengine.io/docs/organisation/scene-management).
```
Comment (davesmith00000) [inline indigo/docs/guides/howto-indigo-game.md:17]: This is fine, but just for future reference there is an internal link format, you just do `[here](scene-management.md)` (I think), and docusaurus does the rest.

File: indigo/docs/guides/howto-indigo-game.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+or [here](https://github.com/PurpleKingdomGames/hello-indigo-sbt) (sbt). In this
+tutorial we're going to take that example and convert it to `IndigoGame`.
+
+The first thing to do here is to make the initial conversion. Change the object declaration like so:
+
+```diff
+ @JSExportTopLevel("IndigoGame")
+-object HelloIndigo extends IndigoSandbox[Unit, Model] {
++object HelloIndigo extends IndigoGame[Unit, Unit, Model, Unit] {
+```
```
Comment (davesmith00000) [inline indigo/docs/guides/howto-indigo-game.md:40]: Much nicer with the diffs 👍

**Review by davesmith00000 — APPROVED** [review]
LGTM!

**Pull request merged**